exporters/etw/test/etw_provider_test.cc

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
And here - same `#ifdef _WIN32` here around the entire file.

Then your code does not have to depend on whether the build system pulls it in or not. The build system can always add it, but it'll be ignored - compiled into an empty object, not linked into final set of tests.